src/redux/selectors/exposedSelectors.js

```diff
@@ -170,6 +172,8 @@ export const getCursorOverlayData = state => state.viewer.cursorOverlay;
 
 export const getOpenElements = state => state.viewer.openElements;
 
+export const getDisabledElements = state => state.viewer.disabledElements;
```

Comment:
As user is able to add more than one custom modal and they are rendered all at the same time, I needed to expose all disabledElements. Previously we had redux selector only to allow predefined modals one by one.